packages/peregrine/lib/talons/Header/useCartTrigger.js

```diff
@@ -9,18 +13,24 @@ export const useCartTrigger = props => {
         mutations: { createCartMutation },
         queries: { getCartDetailsQuery, getItemCountQuery }
     } = props;
+
+    const apolloClient = useApolloClient();
     const [, { toggleDrawer }] = useAppContext();
     const [{ cartId }, { getCartDetails }] = useCartContext();
 
-    const [getItemCount, { data }] = useLazyQuery(getItemCountQuery);
+    const [getItemCount, { data }] = useLazyQuery(getItemCountQuery, {
+        fetchPolicy: 'cache-and-network'
```

Comment:
@sirugh - I couldn't get the CartTrigger to update without a refreshing unless I added this fetch policy. I suspect because the manual cache deletions don't trigger lifecycle events, it was still pulling the value from the cache.